### ISSUE TITLE
libdouble.conversion.so.1 not found

### ISSUE THREAD

**Issue by Catoptrix** [gort818/qtwebflix#32]
Firstly I couldn't start webflix anymore suddenly after 6 months in use. Then I tried to reinstall it with the following error:

`Info: creating stash file /home/obarun/.cache/yay/qtwebflix-git/src/qtwebflix/.qmake.stash
/usr/bin/uic: error while loading shared libraries: libdouble-conversion.so.1: cannot open shared object file: No such file or directory
/usr/bin/uic mainwindow.ui -o ui_mainwindow.h
/usr/bin/uic: error while loading shared libraries: libdouble-conversion.so.1: cannot open shared object file: No such file or directory
make: *** [Makefile:650: ui_mainwindow.h] Error 127
==> ERROR: A failure occurred in build().
    Aborting...
Error making: qtwebflix-git`

**Comment by gort818** [comment]
Looks like libdoubleconversion is provided by qt5-base, if you do not have that installed you can try installing 
> double-conversion

**Comment by omni6** [comment]
yo double post, just wanted to answer in aur-comments.

@Guybrusssh

On my machine qt5webflix build without errors.

`libdouble-conversion.so.*` is part of `double-conversion 3.1.1-1` which is a dependency of `qt5-base` and this should get pulled from `qt5-webengine-widevine` in pkgbuild.
 suppose something is wrong with your system


reinstalling double-conversion can help
and can you look if you have `/usr/lib/libdouble-conversion.so `

**Comment by Catoptrix** [comment]
yeah sorry I posted it everywhere lol, I will link to this when I solved it. Thx for the quick answers ,appreciated.

I have double-conversion 3.1.1-1 installed, reinstalling it didn't solve the problem, qt5-base is installed. 

hmm after checking the path /usr/lib/libdouble-conversion.so
it only finds libdouble-conversion.so.3.0.0
O_o that's weird

it worked flawlessly before for months... O_o what should i do?

**Comment by omni6** [comment]
Mhh.. you can run 'sudo ldconfig' to be safe that all shared libs are correct linked. Otherwise i have no idea...

**Comment by Catoptrix** [comment]
idconfig? i don't have that command. Will try to reinstall double-conversion

**Comment by omni6** [comment]
No not an i... With l (L)

**Comment by Catoptrix** [comment]
ah thx. was running Ldconfig. did nothing. But I think I have an idea what the problem is. I will post again if I know more, it's quite complicated.

**Comment by omni6** [comment]
obarun... well, trivial information.
